src/NuGet.Core/NuGet.ProjectModel/JsonTextReaderExtensions.cs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NuGet.ProjectModel
+{
+    internal static class JsonTextReaderExtensions
+    {
+        internal static IReadOnlyList<string> ReadDelimitedString(this JsonTextReader reader)
+        {
+            if (reader == null)
+            {
+                throw new ArgumentNullException(nameof(reader));
+            }
+
+            if (ReadNextToken(reader))
+            {
+                switch (reader.TokenType)
+                {
+                    case JsonToken.String:
+                        var value = (string)reader.Value;
+
+                        return value.Split(new[] { ' ', ',' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Would it be worth to add the delimiters as params argument?

Reply:
No, not now.  If there's a compelling use case, I'd consider it.